portfolio/src/main/webapp/script.js

```diff
@@ -15,14 +15,29 @@
 /**
  * Adds a random greeting to the page.
  */
-function addRandomGreeting() {
-  const greetings =
-      ['Kayla is 19 years old'];
 
-  // Pick a random greeting.
-  const greeting = greetings[Math.floor(Math.random() * greetings.length)];
 
-  // Add it to the page.
-  const greetingContainer = document.getElementById('greeting-container');
-  greetingContainer.innerText = greeting;
-}
+function KaylaTrivia(){
+   const correct_answers = ['b','a','b','c'] ;
```

Comment:
Missed this on the first pass but in JS you want to name local variables with `lowerCamelCase`: https://google.github.io/styleguide/jsguide.html#naming-local-variable-names